src/Domain/UseCase/VerifyMFA.ts

```diff
@@ -28,7 +28,7 @@ export class VerifyMFA implements UseCaseInterface {
 
     const mfaExtension = await this.itemsRepository.findMFAExtensionByUserUuid(user.uuid)
 
-    if (!mfaExtension) {
+    if (!mfaExtension || !mfaExtension.content) {
```

Comment:
Hmm...what's this checking? If the mfaExtension is deleted? I think that should be the only case content is undefined, in which case it seems like we can check for deleted flag explicitly?

Reply:
True, will fix to utilize `deleted` - will cast the content to string in lines below.